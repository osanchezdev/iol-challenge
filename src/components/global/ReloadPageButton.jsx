import React from 'react';
import Button from 'react-bootstrap/Button';

const ReloadPageButton = () => {
  const reloadPage = () => window.location.reload();

  return (
    <Button variant="secondary" size="lg" onClick={() => reloadPage()}>
      Reload Page
    </Button>
  );
};

export default ReloadPageButton;
