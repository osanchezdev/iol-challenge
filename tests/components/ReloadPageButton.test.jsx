import * as React from 'react';
import {shallow} from 'enzyme';
import ReloadPageButton from '../../src/components/global/ReloadPageButton';

describe('Tests in <ReloadPageButton />', () => {
  test('<ReloadPageButton /> must be rendered correctly', () => {
    const wrapper = shallow(<ReloadPageButton />);

    expect(wrapper).toMatchSnapshot();
  });
});
