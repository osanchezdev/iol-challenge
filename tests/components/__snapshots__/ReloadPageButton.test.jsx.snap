// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <ReloadPageButton /> <ReloadPageButton /> must be rendered correctly 1`] = `
<Button
  active={false}
  disabled={false}
  onClick={[Function]}
  size="lg"
  variant="secondary"
>
  Reload Page
</Button>
`;
